src/components/PaginatedContent.vue

```diff
@@ -109,26 +113,53 @@ const nextLabel = computed(() => {
 })
 
 const handlePrevClicked = () => {
+  setFixedContainerHeight()
   pageIndex.value--
   emit('prev-clicked')
-  scrollToPaginatedContent()
 }
 
 const handleNextClicked = () => {
+  setFixedContainerHeight()
   pageIndex.value++
   emit('next-clicked')
-  scrollToPaginatedContent()
 }
 
-const paginatedContent = ref()
+watch(
```

Comment:
I think we should make a dedicated composable for that because this will be reused in most of the pages containing tables

Reply:
I think now when I fixed the resetting it's not an issue anymore.
The pagination for tables is only handled by PaginatedContent component, right?
If I missed your point, please give me some examples of which other table would you reuse it?

Reply:
I misinterpreted the code. Please ignore it